tst/mockpkg/src/mockpkg.c

```diff
@@ -1,4 +1,4 @@
-#include <gap_all.h>    // GAP headers
+#include "gap_all.h"    // GAP headers
```

Comment:
But the work you are doing here is meant to allow this to be used with `<...>` so I would suggest we should test that here...
```suggestion
#include <gap_all.h>    // GAP headers
```